miner/worker.go

```diff
@@ -1167,7 +1169,10 @@ loop:
 			// Reorg notification data race between the transaction pool and miner, skip account =
 			log.Trace("Skipping account with hight nonce", "sender", from, "nonce", tx.Nonce())
 			txs.Pop()
-
+		case errors.As(err, &errL1):
+			// Skip the current transaction failed on L1Sload precompile with L1RpcError without shifting in the next from the account
```

Comment:
Better to also mention that this tx will be left in the txpool and automatically retried for future blocks. So eventually (once the connection to the L1 node recovers) we should be able to process it.